### PR TITLE
e2e build and test pipelne

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@stevekuznetsov

File: cluster/ci/origin/pipelines/e2e/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              podAnnotation(key: "alpha.image.policy.openshift.io/resolve-names", value: "*")
+            ],
+            volumes: [
+              secretVolume (
+                secretName: "${this._buildName}",
+                mountPath: '/var/secrets/kubeconfig'
+              )
+            ]
+          ) {
+            node("origin-e2e") {
```
Comment (stevekuznetsov) [inline cluster/ci/origin/pipelines/e2e/Jenkinsfile]: remember to upload artifacts

File: cluster/ci/origin/pipelines/e2e/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            [$class: "StringParameterValue", name: "PULL_BASE_SHA", value: params.PULL_BASE_SHA],
+            [$class: "StringParameterValue", name: "PULL_REFS", value: params.PULL_REFS],
+            [$class: "StringParameterValue", name: "PULL_NUMBER", value: params.PULL_NUMBER],
+            [$class: "StringParameterValue", name: "PULL_PULL_SHA", value: params.PULL_PULL_SHA],
+          ]
+          build job: jobName, parameters: buildParams, wait: true
+        }
+      }
+    }
+    stage("Run e2e test") {
```
Comment (stevekuznetsov) [inline cluster/ci/origin/pipelines/e2e/Jenkinsfile]: //TODO instrument a cluster debug thing for post-run to get logs, events, etc

**Comment by csrwng** [comment]
Thanks @stevekuznetsov. Uploading artifacts implemented

**Comment by stevekuznetsov** [comment]
/lgtm 
with the stashing

**Comment by csrwng** [comment]
TODO for image build: 
Automatically generate the builds from hack/lib/constants.sh in origin

**Pull request merged**